include/openssl/bio.h

```diff
@@ -144,6 +144,10 @@ extern "C" {
 # endif
 
 # define BIO_CTRL_DGRAM_SET_PEEK_MODE      71
+# define BIO_CTRL_SET_OFFLOAD_TX               72
```

Comment:
the name "offload tx" seems strange to me.  is it a kernel kind of usage?  I would see it as just offload without the tx.  here and in the next few.

Reply:
No, it is not a kernel kind of usage. Maybe BIO_CTRL_SET_KTLS or BIO_CTRL_SET_KTLS_TX is a better name?
The "TX" part is related to the fact that this is directional - only the transmit side is supported at the moment. A future patchset will add the receive side as well. Should we assume that the direction is implied by the surrounding code and call this just set_ktls?

Reply:
I like just KTLS.

Reply:
Done. There is only one place where the direction is still required - BIO_set_ktls. For this purposes a new "is_tx" parameter is added to this macro.